site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -190,7 +190,11 @@ To upgrade FusionAuth when running with `docker-compose`:
 . Start it up: `docker-compose up`.
 . Login to the administrative UI. 
 
-If there were migrations required, they'll be applied automatically to your database if the installation link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. If the installation is in production mode, make sure you apply the migrations out of band. 
+If there were migrations required, they'll be applied automatically to your database if the installation link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. If the installation is in production mode and you run a version before 1.19, make sure you apply the migrations out of band. 
+
+If you run version 1.19 or later, you can use silent mode for db upgrades even in production. Prior to this version we didn't have a way to lock a db across multiple nodes in a way to ensure only one node attempted the upgrade. Read the /docs/v1/tech/guides/silent-mode/[silent mode guide] for more information.
```

Comment:
I may just ditch `Prior to this version we didn't have a way to lock a db across multiple nodes in a way to ensure only one node attempted the upgrade.` not sure if that is relevant.